packages/jellyfish-wallet-encrypted/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "private": false,
+  "name": "@defichain/jellyfish-wallet-encrypted",
+  "version": "0.0.0",
+  "description": "A collection of TypeScript + JavaScript tools and libraries for DeFi Blockchain developers to build decentralized finance on Bitcoin",
+  "keywords": [
+    "DeFiChain",
+    "DeFi",
+    "Blockchain",
+    "API",
+    "Bitcoin"
+  ],
+  "repository": "DeFiCh/jellyfish",
+  "bugs": "https://github.com/DeFiCh/jellyfish/issues",
+  "license": "MIT",
+  "contributors": [
+    {
+      "name": "DeFiChain Foundation",
+      "email": "engineering@defichain.com",
+      "url": "https://defichain.com/"
+    },
+    {
+      "name": "DeFi Blockchain Contributors"
+    },
+    {
+      "name": "DeFi Jellyfish Contributors"
+    }
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsc"
+  },
+  "dependencies": {
+    "@defichain/jellyfish-crypto": "0.0.0",
+    "@defichain/jellyfish-wallet": "0.0.0",
+    "@defichain/jellyfish-wallet-mnemonic": "0.0.0",
+    "bignumber.js": "^9.0.1",
+    "bs58": "^4.0.1",
+    "scryptsy": "^2.1.0",
+    "tiny-secp256k1": "^2.0.0"
```

Comment:
@fuxingloh any idea what's wrong here? The GH action test cannot import this but it works in my machine, `npm run all` passed

Reply:
Fixed. Should be messed up npm cached hash tree due to accidentally ran npm i using older npm ver. Delete node_module, revert package-lock, force clean npm cache and reinstall fix it.